toolkit/committee-selection/pallet/src/lib.rs

```diff
@@ -68,17 +75,20 @@ pub mod pallet {
 			+ From<u64>
 			+ Into<u64>;
 
+		/// Type of committee members returned by `select_authorities`.
 		type CommitteeMember: Parameter
 			+ Member
 			+ MaybeSerializeDeserialize
 			+ MaxEncodedLen
 			+ CommitteeMember<AuthorityId = Self::AuthorityId, AuthorityKeys = Self::AuthorityKeys>;
 
+		/// Selects a committee for `sidechain_epoch` based on selection inputs `input`.
```

Comment:
```suggestion
		/// Should select a committee for `sidechain_epoch` based on selection inputs `input`. Should return None if selection was impossible for the given input.
```
I reworded to "should", because this defines requirements that runtime have to satisfy.

Reply:
Thank you, originally I've done the same but I looked up other api declarations for reference and they didn't follow this. I'll go back and fix those too.